src/Project.js

```diff
@@ -601,38 +635,17 @@ class Project extends DirItem {
             });
         }
 
-        const fmt = new Intl.NumberFormat("en-US", {
-            maxFractionDigits: 2
-        });
-        logger.info(`Total Elapse Time: ${String(totalTime)} seconds`);
-        logger.info(`                             ${`Average over`.padEnd(15, ' ')}${`Average over`.padEnd(15, ' ')}${`Average over`.padEnd(15, ' ')}`);
-        logger.info(`                   Total     ${`${String(this.fileStats.files)} Files`.padEnd(15, ' ')}${`${String(this.fileStats.modules)} Modules`.padEnd(15, ' ')}${`${String(this.fileStats.lines)} Lines`.padEnd(15, ' ')}`);
-        if (results.length) {
-            logger.info(
-                    `Errors:            ${String(this.resultStats.errors).padEnd(10, ' ')}${fmt.format(this.resultStats.errors/this.fileStats.files).padEnd(15, ' ')}${fmt.format(this.resultStats.errors/this.fileStats.modules).padEnd(15, ' ')}${fmt.format(this.resultStats.errors/this.fileStats.lines).padEnd(15, ' ')}`);
-            if (!this.options.errorsOnly) {
-                logger.info(
-                    `Warnings:          ${String(this.resultStats.warnings).padEnd(10, ' ')}${fmt.format(this.resultStats.warnings/this.fileStats.files).padEnd(15, ' ')}${fmt.format(this.resultStats.warnings/this.fileStats.modules).padEnd(15, ' ')}${fmt.format(this.resultStats.warnings/this.fileStats.lines).padEnd(15, ' ')}`);
-                logger.info(
-                    `Suggestions:       ${String(this.resultStats.suggestions).padEnd(10, ' ')}${fmt.format(this.resultStats.suggestions/this.fileStats.files).padEnd(15, ' ')}${fmt.format(this.resultStats.suggestions/this.fileStats.modules).padEnd(15, ' ')}${fmt.format(this.resultStats.suggestions/this.fileStats.lines).padEnd(15, ' ')}`);
+        if (this.options.opt.output) {
+            if (typeof resultFull !== 'undefined') {
+                let file = this.options.opt.output;
+                let fileDir = path.dirname(file);
+                if (!fs.existsSync(fileDir)) {
+                    fs.mkdirSync(fileDir);
+                }
+                fs.writeFileSync(file, resultFull, "utf8");
             }
         }
```

Comment:
else output the restFull string to the logger. If you don't do that, then you will get no output at all if you have a formatter with the formatOutput method defined, but you do not choose to direct the output to a file.

Reply:
I update it.